TRAVIS_WEBHOOK_README.md

```diff
@@ -1,18 +1,14 @@
 # Travis Webhook Instructions
 
-## Make sure travis is installed with gem
+## Make sure travis is installed with Homebrew
```

Comment:
Does this require the Homebrew version? It's only for macOS/Linux.

Reply:
Updated to use either. gem wasn't working correctly for me.

https://github.com/Esri/calcite-components/blob/master/TRAVIS_WEBHOOK_README.md